prov/rxm/src/rxm_init.c

```diff
@@ -412,6 +414,10 @@ RXM_INI
 			"decrease noise during cq polling, but may result in "
 			"longer connection establishment times. (default: 10000).");
 
+	fi_param_define(&rxm_prov, "data_progress_auto", FI_PARAM_BOOL,
+			"Force data auto-progress even if app requested manual "
+			"progress (default: 0) \n");
```

Comment:
default: false or no?

Reply:
Default is false/0/no, that is auto-progress is not forced.

Reply:
I wasn't clear.  I was referring to the wording in the message to state the default in terms of "false" or "no" instead of an integer 0.

Reply:
Got it, I shall update that.